src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -161,8 +161,13 @@ export class OpenSheetMusicDisplay {
         this.drawer.clear();
         this.drawer.resize(width, height);
         this.drawer.scale(this.zoom);
+
+        // clear and fill with background color
+        this.drawer.clear(0, 0, width, height);
```

Comment:
There are 2 clears. Why?

Reply:
the first clear actually only removes the SVG children in the SVG backend in this case. I left it in in case the resizing and scaling is slower if we don't remove the SVG children beforehand.
So, clear without arguments behaves like before, clear with position and dimensions makes a colored background (rectangle).
We could separate these methods, of course. But i don't think removing children from a list or drawing a single rectangle costs much performance.